doc/content/guides/connecting-gateways/kerlinkwirnetistation/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Kerlink Wirnet iStation"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-istation/). This page guides you to connect it to {{% tts %}}.
```

Comment:
Isn't this the iStation then?

Isn't this extremely confusing to have Station and iStation?

Reply:
> Isn't this the iStation then?

It is indeed. Will fix.

> Isn't this extremely confusing to have Station and iStation?

It is indeed. I doubt Kerlink will rename the gateway though, so we have to deal with it.